frontend/src/components/Sheet/Sheet.module.scss

```diff
@@ -0,0 +1,80 @@
+@import "../../variables";
+
+.containerLogin {
+  height: 100vh;
+  width: 100vw;
+  opacity: 95%;
+  background: rgba(0, 0, 0, 0.2);
```

Comment:
like i mightve said elsewhere using rgba at the same time as opacity doesnt make a ton of sense imo unless you're trying to make the contents see thru?